lib/util/utils.js

```diff
@@ -535,6 +535,34 @@ class Utils {
       return prev;
     }, url);
   }
+
+  /**
+   * Collects all unquoted arguments in an array of arguments and returns them.
+   * e.g.: [--argument="Hello there"] will be okay.
+   * e.g.: [--argument=Hello there] will be collected because it contains an unquoted whitespace.
+   * @param args
+   * @returns {Array}
+   */
+  static findUnquotedCliArgumentsWithWhitespaces(args) {
+
```

Comment:
I think you want to remove this blank line here; it doesn't match the style of the rest of the file.

Reply:
done :)